builds/understanding-builds.adoc

```diff
@@ -0,0 +1,40 @@
+//include:modules/common-attributes.adoc[]
+// This assembly is included in the following assemblies:
+//
+// * assembly/understanding-builds
+
+// Retains the context of the parent assembly if this assembly is nested within another assembly.
+// For more information about nesting assemblies, see: https://redhat-documentation.github.io/modular-docs/#nesting-assemblies
+// See also the complementary step on the last line of this file.
+ifdef::context[:parent-context: {context}]
+
+
+:context: builds
+[id='builds_{context}']
+= Understanding builds
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+toc::[]
+
+The following sections define the concept of a build, and outline the primary
```

Comment:
Distinguish that this section is OpenShift specific (not Kube as deployments are)